OsmAnd/res/values/strings.xml

```diff
@@ -63,32 +63,32 @@
     <string name="weather_pressure_hectopascals">Hectopascals</string>
     <string name="weather_pressure_inches_of_mercury">Inches of mercury</string>
     <string name="weather_pressure_millimeters_of_mercury">Millimeters of mercury</string>
-    <string name="weather_prefs_descr">24h hour &amp; 7 day forecast, updated every 3 hours</string>
+    <string name="weather_prefs_descr">Daily forecast every 24 hours and 7-day forecast every 3 days</string>
     <string name="weather_temperature_celsius">Celsius</string>
     <string name="weather_temperature_fahrenheit">Fahrenheit</string>
-    <string name="action_not_supported_with_map_style">Action is not supported by selected map style.</string>
-    <string name="rendering_engine_failed_descr">OsmAnd has reverted to \'Version 1\' map rendering. You can try to switch back to the new \'Version 2 (OpenGL)\' rendering engine in the Settings.\n\nPlease support improvements by providing your error log.</string>
-    <string name="rendering_engine_failed">OsmAnd\'s map rendering has recently crashed.</string>
+    <string name="action_not_supported_with_map_style">Pick a different map style to use this.</string>
+    <string name="rendering_engine_failed_descr">Reverted to \'Version 1\' map rendering. Try switching back to the newer \'Version 2 (OpenGL)\' in the settings.\n\nProviding your error log can help improve the rendering engine.</string>
+    <string name="rendering_engine_failed">OsmAnd\'s map rendering crashed.</string>
     <string name="share_crash_log">Share crash log</string>
     <string name="dismiss_changes_descr">All changes will be dismissed.</string>
     <string name="dismiss_changes">Dismiss changes?</string>
     <string name="restart_now">Restart now</string>
     <string name="map_rendering_engine_v1">Version 1</string>
     <string name="map_rendering_engine_v2">Version 2 (OpenGL)</string>
-    <string name="map_rendering_engine_descr">Version 2 (OpenGL) is the faster map rendering engine also featuring 2.5D view. Version 1 rendering was the default before OsmAnd 4.3, and is still preferable for older devices.</string>
+    <string name="map_rendering_engine_descr">Version 2 (OpenGL) is faster and also features a 2.5D view. Version 1 was the default before OsmAnd 4.3, and is still preferable for older devices.</string>
     <string name="map_rendering_engine">Map rendering engine</string>
     <string name="show_borders_of_downloaded_maps">Show borders of downloaded maps</string>
     <string name="show_map_symbols">Show map symbols</string>
     <string name="show_map_symbols_desc">Text, road signs, and others.</string>
     <string name="purchases_feature_desc_external_sensors">Display and record data from external sensors: bicycle speed, bicycle power, cadence, and heart rate.</string>
-    <string name="external_sensors_support">External sensors support</string>
+    <string name="external_sensors_support">External sensors</string>
```

Comment:
What's wrong with 'support' here, have you checked the context?

Reply:
It is redundant
Like ~"External standalone I/O sensor input support" or similar.

Reply:
It is consistent nonsense :)